app/code/Magento/Customer/Test/Unit/Model/Metadata/Form/MultiselectTest.php

```diff
@@ -143,7 +144,6 @@ public function outputValueTextDataProvider()
     {
         return [
             'empty' => ['', ''],
-            'null' => [null, ''],
```

Comment:
why was it removed?

Reply:
@ihor-sviziev I think we can't get null value in Multiselect. What do you think?

Reply:
So why it was possible in past?

Reply:
@ihor-sviziev hello! With null value, this test will fail in [Multiselect.php](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Customer/Model/Metadata/Form/Multiselect.php#L49) in PHP 8.1. The method `outputValue` overwrites the method from AbstractData which has a parameter only string or array.
We can change the test, or change a model. What do you think about this?

Reply:
Lets use `false` instead of `null`
https://github.com/magento/magento2/blob/203a44f9e755fa6d2e057f1b99efbaff17546a80/app/code/Magento/Customer/Model/Metadata/ElementFactory.php#L61

Reply:
Hello @xmav, thanks for your review, I fixed code by your recommendation, please check if everything is well.

Thank you